src/pages/landing/stocks/_dtrading.js

```diff
@@ -0,0 +1,162 @@
+    @media ${device.tabletL} {
+        padding: 17px 0 4px 0;
+    }
+`
+const Content = styled.div`
+    width: 60%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-right: ${(props) => props.margin_right};
+    margin-left: ${(props) => props.margin_left};
+
+    ${Text} {
+        margin-top: 2.2rem;
+        font-size: 2.4rem;
+        @media ${device.laptopM} {
+            font-size: 2.4rem;
+        }
+        @media ${device.tabletL} {
+            font-size: 1.6rem;
+            text-align: center;
+        }
+    }
+
+    @media ${device.tabletL} {
+        width: 100%;
+        max-width: 58.8rem;
+        margin: 0 auto;
+    }
+`
+
+const ImageWrapper = styled.div`
+    display: flex;
+    width: 40%;
+    margin-right: ${(props) => props.margin_right};
+
+    @media ${device.tabletL} {
+        margin: 2rem auto;
+        max-width: 58.8rem;
+        width: 100%;
+    }
+`
+const StyledHeader = styled(Header)`
+    line-height: 1.25;
+
+    @media ${device.tabletL} {
+        font-size: 40px;
+        line-height: 40px;
+        margin-top: 2rem;
+        text-align: center;
+    }
+`
+const Row = styled.div`
+    flex-direction: ${(props) => props.flex_direction};
+    width: 100%;
+    display: flex;
+    margin-top: 4rem;
+
+    &:first-child {
+        margin-top: 0;
+    }
+
+    @media ${device.tabletL} {
+        flex-direction: column;
+    }
+`
+const query = graphql`
+    query {
+        dbot_strategy: file(relativePath: { eq: "dbot-strategy.png" }) {
+            ...fadeIn
+        }
+        dbot_build_strategy: file(relativePath: { eq: "dbot-build-strategy.png" }) {
+            ...fadeIn
+        }
+        dbot_maximise_profits: file(relativePath: { eq: "dbot-maximise-profits.png" }) {
+            ...fadeIn
+        }
+        dbot_track_your_performance: file(relativePath: { eq: "dbot-track-your-performance.png" }) {
+            ...fadeIn
+        }
+        dbot_get_integrated_help: file(relativePath: { eq: "dbot-get-integrated-help.png" }) {
+            ...fadeIn
+        }
+        dbot_save_your_strategies: file(relativePath: { eq: "dbot-save-your-strategies.png" }) {
+            ...fadeIn
+        }
+        dmt_5_ipad_iphone: file(relativePath: { eq: "dmt-5-ipad-iphone.png" }) {
+            ...fadeIn
+        }
+        dmt_5_mac: file(relativePath: { eq: "dmt-5-mac.png" }) {
+            ...fadeIn
+        }
+        make_smarter: file(relativePath: { eq: "make-smarter.png" }) {
+            ...fadeIn
+        }
+        trade_what_you_want: file(relativePath: { eq: "trade-what-you-want.png" }) {
+            ...fadeIn
+        }
+        how_you_want: file(relativePath: { eq: "how-you-want.png" }) {
+            ...fadeIn
+        }
+        stocks_zero_fees: file(relativePath: { eq: "stocks-zero-fees.png" }) {
+            ...fadeIn
+        }
+        stocks_blue_chip: file(relativePath: { eq: "stocks-blue-chip.png" }) {
+            ...fadeIn
```

Comment:
Can we create a separate folder for all these `png` called `stock-indices`? This will reduce the number of files in the main `common` folder.